control_panel_api/views.py

```diff
@@ -183,6 +183,7 @@ class S3BucketViewSet(viewsets.ModelViewSet):
     serializer_class = S3BucketSerializer
     filter_backends = (S3BucketFilter,)
     permission_classes = (S3BucketPermissions,)
+    filter_fields = ('is_data_warehouse',)
```

Comment:
Should we have a test for the filtering by `is_data_warehouse` flag?

Reply:
Sorry, I didn't see you added a test in the existing `test_list` 👍

Reply:
😄